frontend/src/api/problem.ts

```diff
@@ -10,7 +10,5 @@ const routes = {
   postProblem: `${basePath}/`,
 };
 
-const getProblems = (): Promise<Problem[]> => fetch(routes.getProblems)
+export const getProblems = (): Promise<Problem[]> => fetch(routes.getProblems)
```

Comment:
Note: No action required, but I'd like to get input on this; I think we determined last time that the `Problem[]` type doesn't actually type-check the function correctly. If this is the case, should we try to fix it, or is it not a big deal?

Reply:
I think it's fine to leave as is

Reply:
To update on this, I think [this link](https://stackoverflow.com/questions/55960602/how-to-cast-fetch-api-response-to-a-specific-type-in-typescript) explains the behavior well; basically, TypeScript can't actually validate the API responses match the defined types (cause it comes in at runtime), so it's just up to us to make sure it matches properly (so that at compile time in your IDE TypeScript can work properly when you use those types elsewhere in the app).